rcloud.support/R/rcloud.support.R

```diff
@@ -344,6 +344,9 @@ rcloud.update.notebook <- function(id, content, is.current = TRUE) {
         content$files <- cfiles
     }
     content <- .gist.binary.process.outgoing(id, content)
+
+    # save thumbnails to key-value database
+    # rcloud.set.thumb(id = id, thumb_png = content$files$thumb.png.b64$content)

```

Comment:
accidental comment here, fixed in b77fecbf94

also it looks like you used a windows machine for this commit and it added some messy line endings, please take a look at https://help.github.com/articles/dealing-with-line-endings/#platform-windows